web/themes/custom/cloudy/templates/media/media--document.html.twig

```diff
@@ -13,7 +13,8 @@
  */
 #}
 {% embed 'media.html.twig' with {
-  'margin' : '0'
+  'margin' : '0',
+  'classes': content.field_is_accessible['#items'].0.value ? ['skip-editoria11y-check'],
```

Comment:
Could you also try applying my suggested changes to `media.html.twig` so that the additional classes get merged with the default classes?

https://github.com/eGovPDX/portlandor/pull/3068#discussion_r2047516682

Otherwise, the `skip-editoria11y-check` overrides all of the other classes, some of which apply CSS styles to the media.

![image](https://github.com/user-attachments/assets/b387e717-3cd3-4168-ab9e-f445fcc07f82)

Reply:
Missed this update when switching back and forth between stories. Suggested fix applied.